### PR TITLE
LPS-191284 | EVP Automated Test | master 

### PR DESCRIPTION
**Pull request by epark18**
https://liferay.atlassian.net/browse/LPS-191284

### CODE REVIEW

**Comment by epark18** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPHomePage.testcase [hunk 1]
```diff
@@ -65,6 +65,24 @@ definition {
 		}
 	}
 
+	@description = "LPS-172661 - Verify clicking on News title will open a modal to display content."
+	@priority = 3
+	test CanSeeWholeNews {
+		task ("Given a user navigates to the EVP home page") {
+			EVPUtils.gotoEVP();
+		}
```
Comment (MGabriellaS) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPHomePage.testcase:73]: Hey @epark18 , I think here you could login as a employee to assert that employee user can see all the news content. You can add this code: `User.logoutAndLoginPG( userLoginEmailAddress = "employee@liferay.com", userLoginFullName = "Employee User");` before `EVPUtils.gotoEVP();` So you can also change the task to:`"Given a Employee user navigates to the EVP home page"`

**Comment by epark18** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPHomePage.testcase (hunk 1 above)
Comment (epark18) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPHomePage.testcase:77]: Thank you @MGabriellaS I made the change.

**Comment by MGabriellaS** [comment]
The code looks good to me!

**Comment by MGabriellaS** [comment]
ci:forward

**Pull request closed without merging**